lib/FusionInventory/Agent/Task/Inventory/Generic/Dmidecode/Memory.pm

```diff
@@ -103,13 +103,37 @@ sub _getMemories {
                 TYPE             => $info->{'Type'},
                 SERIALNUMBER     => $info->{'Serial Number'},
                 MEMORYCORRECTION => $infos->{16}[0]{'Error Correction Type'},
-                MANUFACTURER     => $manufacturer
+                MANUFACTURER     => undef,
```

Comment:
Here you can just remove MANUFACTURER if you want to set it later.

Reply:
I could have, but this would've lead to modifying existing test files much more. Do you still want me to remove?

Reply:
Not necessary then, this is definitively not important.